index.html

```diff
@@ -5,8 +5,98 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet">
 </head>
 <body>
-    
+    <main class="container">
+        <header class="header">
+            <h1> Weather Report<h1>
+            <span class="city-header"> For the lovely city of
+                <!-- ✨ <span id="city-name">Some city</span> ✨    -->
```

Comment:
Style: It's an industry best practice to remove commented out code before submitting a PR.